com.unity.formats.alembic/Documentation~/index.md

```diff
@@ -17,33 +20,36 @@ The Alembic package supports these features:
 
 > **Note:** If you need to use the Alembic Clip Recorder feature, you must also install the [Unity Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@latest/index.html) package (minimum version: 2.0.0).
 
-## Requirements
 
-The Alembic Package is compatible with Unity Editor 2018.1 and above.
+## Package technical details
 
-The package is available on these 64-bit desktop platforms:
-* Microsoft® Windows® 10
-* macOS® Sierra (10.12)
-* GNU/Linux (CentOS 7, Ubuntu 16.x and Ubuntu 17.x)
+### Preview package
 
-## Known limitations
+This package is in preview, so it is not ready for production use. The features and documentation in this package might change before it is verified for release.
```

Comment:
The package will become re-verified briefly (alembic is already verified on a previous version). I would drop the sentence.